packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -156,10 +159,6 @@ export class LocalChannelContext implements IChannelContext {
         this.channel = channel;
         this._isLoaded = true;
 
-        if (this.attached) {
```

Comment:
Can you please let me know why this code is not needed any more? Can't figure it out

Reply:
Actually this was redundant because during load we call factory.load which actually assign services and attach delta handler. So we don't need to do it again. I added the comment on factory.load about it in the PR.